exporter/otlp-http/CHANGELOG.md

```diff
@@ -0,0 +1,141 @@
+# Release History: opentelemetry-exporter-otlp
```

Comment:
Same thing here - let's start with a fresh changelog whose first entry refers to the old one.